src/less/icons.less

```diff
@@ -28,7 +33,7 @@
 @cf-icon-prefix:                cf-icon;
 @cf-icon-path:                  '../fonts';
 @cf-icon-ie7-support:           true;
-
+@cf-border-color:               #eee;
```

Comment:
I wonder if this could use a different name? It seems like existing CF variables don't include the `cf-` prefix alone generally, e.g. https://github.com/cfpb/cf-typography/blob/gh-pages/src/cf-typography.less#L30, https://github.com/cfpb/cf-core/blob/gh-pages/src/cf-vars.less#L39, https://github.com/cfpb/cf-layout/blob/gh-pages/src/cf-layout.less#L22. Perhaps normalize it to the other variable names in this project, e.g. `cf-icon-border-color`.